master/reference/calicoctl/resources/felixconfig.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Felix Configuration Resource (FelixConfiguration)
+---
+
+A Felix configuration resource (FelixConfiguration) represents Felix configuration options for the cluster.
+
+For `calicoctl` commands that specify a resource type on the CLI, the following
+aliases are supported (all case insensitive): `felixconfiguration`, `felixconfig`, `felixconfigurations`, `felixconfigs`.
+
+See [Configuring Felix]({{site.baseurl}}/{{page.version}}/referance/felix/configuration) for more details.
+
+### Sample YAML
+
+```yaml
+apiVersion: projectcalico.org/v2
+kind: FelixConfiguration
+metadata:
+  name: default
+spec:
+  ipv6Support: false
+  ignoreLooseRPF: true
+  chainInsertMode: Append
+```
+
+### BGP Peer Definition
+
+#### Metadata
+
+| Field       | Description                 | Accepted Values   | Schema |
+|-------------|-----------------------------|-------------------|--------|
+| name     | Unique name to describe this resource instance. Must be specified. | Alphanumeric string with optional `.`, `_`, `-`, or `/` | string |
+
+- Resource with the name "default" has a specific meaning - this contains the Felix global default configuration overrides.
+- Resource with the name "node.<nodename>" has the specific meaning that this contains the node-specific overrides, and will be applied to the node <nodename>.  When deleting a node the FelixConfiguration resource associated with the node will also be deleted.
+
+#### Spec
+
+| Field       | Description                 | Accepted Values   | Schema | Default    |
+|-------------|-----------------------------|-------------------|--------|------------|
+| logFilePath | The full path to the Felix log. Set to `none` to disable file logging. | string | string | `/var/log/calico/felix.log` |
+| logSeveritySys | The log severity above which logs are sent to the syslog. Set to `NONE` for no logging to syslog. | DEBUG, INFO, WARNING, ERROR, CRITICAL, or NONE (case-insensitive) | string | `INFO` |
+| logSeverityFile| The log severity above which logs are sent to the log file. | Same as `logSeveritySys` | string | `INFO` |
+| logSeverityScreen | The log severity above which logs are sent to the stdout. | Same as LogSeveritySys | string | `INFO` |
+| startupCleanupDelay | Delay, in seconds, before Felix does its start-of-day cleanup to remove orphaned iptables chains and ipsets.  Before the first cleanup, Felix operates in "graceful restart" mode, during which it preserves any pre-existing chains and ipsets. In a large deployment you may want to increase this value to give Felix more time to load the initial snapshot from etcd before cleaning up. | int | int | `30` |
```

Comment:
You might also want to consider this PR https://github.com/projectcalico/calico/pull/1125, which I am trying to get merged....just need one technical reviewer's approval there but I am not sure who is the best person with @fasaxc being the Felix expert.

Reply:
Good catch, I didn't know about that. I'll update my PR based on that one.